addons/l10n_pt_pos/static/src/xml/OrderReceipt.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-inherit="point_of_sale.OrderReceipt" t-inherit-mode="extension" owl="1">
+        <xpath expr="//div[hasclass('before-footer')]" position="after">
+            <div t-if="receipt.isCountryPortugal" class="receipt-inalterable-hash pos-receipt-center-align">
+                <div class="pos-receipt-order-data">
+                    <t t-if="!receipt.l10nPtPosInalterableHash">
+                        Este documento não serve de fatura<br/>
+                        Emitido por programa certificado n.º TODO/AT
```

Comment:
Just leaving a mark to avoid forgetting this TODO (and the one a few lines below)